OP2Utility.h

```diff
@@ -1,7 +1,7 @@
 #pragma once
 
 #include "StringHelper.h"
-#include "StreamReader.h"
+#include "Streams/StreamReader.h"
```

Comment:
What is the purpose of OP2Utility.h? Is this a generic import header to bring in everything the library has to offer, or does it have more limited project specific usage? If it's a generic import header, then it should probably import everything. If not, I assume it's correct as is.